example/ionic-angular-v2/src/app/app.module.ts

```diff
@@ -0,0 +1,56 @@
+import { NgModule, ErrorHandler } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { RouteReuseStrategy } from '@angular/router';
+
+import { IonicModule, IonicRouteStrategy } from '@ionic/angular';
+import { SplashScreen } from '@ionic-native/splash-screen/ngx';
+import { StatusBar } from '@ionic-native/status-bar/ngx';
+
+import * as Sentry from '@sentry/capacitor';
+import { init as sentryAngularInit, createErrorHandler } from '@sentry/angular';
+import { Integrations } from '@sentry/tracing';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+
+// ATTENTION: Change the DSN below with your own to see the events in Sentry. Get one at sentry.io
+Sentry.init(
+  {
+    dsn:
+      'https://4079af8b316240ea9453eb0a23b715cc@o447951.ingest.sentry.io/5522756',
+    // An array of strings or regexps that'll be used to ignore specific errors based on their type/message
+    ignoreErrors: [/MiddleEarth_\d\d/, 'RangeError'],
+    // Debug mode with valuable initialization/lifecycle information
```

Comment:
What about something like this?
```suggestion
    // To see what the Sentry SDK is doing; Helps when setting things up
```